code/__DEFINES/atmospherics.dm

```diff
@@ -535,6 +535,18 @@
 		out_var += cached_gases[total_moles_id][MOLES];\
 	}
 
+GLOBAL_LIST_INIT(nonoverlaying_gases, typecache_of_gases_with_no_overlays())
+#define GAS_OVERLAYS(gases, out_var)\
```

Comment:
Can you rename this something more obvious that its mutating another variable? i.e. `GAS_OVERLAYS_INTO`? Just a nitpick, but I think it makes it more obvious from the code that something is actually happening here.

Reply:
They're all like that, it's for consistency with the procs that wrap them 
![image](https://user-images.githubusercontent.com/58055496/116769799-b1332c00-a9f3-11eb-8f88-f0cedd22a6fb.png)

Reply:
ugh